Nos/Views/ProfileEdit/ProfileEditView.swift

```diff
@@ -40,10 +47,45 @@ struct ProfileEditView: View {
     }
 
     var body: some View {
+        let avatarSize: CGFloat = 99
+        
         NosForm {
-            AvatarView(imageUrl: URL(string: avatarText), size: 99)
+            ImagePickerButton(cameraDevice: .front) { imageURL in
+                Task { await uploadItem(at: imageURL) }
+            } label: {
+                ZStack {
+                    if let selectedPhoto {
+                        ZStack {
+                            selectedPhoto
+                                .resizable()
+                                .renderingMode(.original)
+                                .aspectRatio(contentMode: .fill)
+                                .clipShape(Circle())
+                            
+                            if isUploadingPhoto {
+                                ProgressView()
+                                    .scaleEffect(1.5)
+                            }
+                        }
+                        .disabled(isUploadingPhoto)
+                    } else {
+                        AvatarView(imageUrl: URL(string: avatarText), size: avatarSize)
+                    }
+                    
+                    HStack {
+                        Spacer()
+                        
+                        VStack {
+                            Spacer()
+                            
+                            Image.editButton
```

Comment:
This button doesn't look like it's quite far enough in the corner. In Figma the bottom of this button aligns with the bottom of the AvatarView, but on this branch there is some margin there. Not sure where it's coming from. Maybe we need `(spacing: 0)` after the `HStack` and `VStack`?

Reply:
This is happening because the asset itself has some small insets. You can see it on the top and leading edges if you look at the svg in Finder. An `offset(x:y:)` modifier fixes it up though. Alternatively we could ask Sebastian for one without that.  

<img width="173" alt="edit-button_imageset" src="https://github.com/user-attachments/assets/916a707c-4d8f-46ae-9d38-2247f1cfd187">